esp/templates/program/modules/onsiteattendance/attendance.html

```diff
@@ -5,6 +5,12 @@
 {% block stylesheets %}
     {{ block.super }}
     <link rel="stylesheet" href="/media/styles/forms.css" type="text/css" />
+    <link rel="stylesheet" href="/media/styles/expand_display.css" type="text/css" />
+{% endblock %}
+
+{% block xtrajs %}
+    {{ block.super }}
+    <script src="/media/scripts/sorttable.js"></script>
```

Comment:
maybe add a note to the top of student attendance page after the explanation saying "All tables are sortable by the columns if you click on the headers." since its not super obvious that this is a feature but it is very cool and helpful